tests/integration/cloud/clouds/test_vmware.py

```diff
@@ -93,3 +93,23 @@ def test_verify_ssl_false(self):
         # check if instance returned with salt installed
         self.assertInstanceExists(ret_val)
         self.assertDestroyInstance()
+
+    def test_instant_clone(self):
+        """
+        Tests creating Instant Clone VM
+        """
+        # salt-cloud -p my-instant-clone IC3
+        profile_name = "my-instant-clone"
```

Comment:
We don't have cloud tests run on PRs, so let me test this to make sure it passes. I'll try this out tomorrow.

Reply:
Cool

Reply:
I'm still working on getting access here. will update when i've tested.

Reply:
No issues

Reply:
I'm probably doing something wrong in my setup, but I'm seeing this error on creation  of the VM:

```
message = "Module 'GuestOS' power on failed. "
         message = 'The configuration file does not specify a guest operating system. Select a guest operating system from the General p
age on the Options tab of Virtual Machine Settings. '
      },

```

Any ideas what I'm missing?

Reply:
The parent VM creation is failing ryt?
some issue might be with the image you are specifying
can you send the profile configuration file I can try on my setup?
https://docs.saltproject.io/en/latest/topics/cloud/vmware.html#creating-a-vm

Reply:
```
vmware-test:
  provider: vmware-config
  instantclone: vmname
  num_cpus: 1
  memory: 2GB
  cluster: cluster_name
  datastore: datastore_name
  datacenter: datacenter
  host: hostname
  ssh_username: sshuser
  password: password
```

The VM is actually created, I just see that error on creation. It might not be an issue and just related to how my VM is setup.

Reply:
Okay cool

Reply:
@Ch3LL So is this getting instant cloned?

Reply:
Yes its getting cloned I just saw that error in teh logs when it occurred so wanted to confirm if it was an issue or not.

Reply:
OK 👍

Reply:
If we get it merged in the next coming weeks we can get it into the next feature release (Silicone)

I went ahead and add this PR https://github.com/saltstack/salt/pull/60383  which will run the cloud tests you added. If those pass we can get this merged in. I think with my testing setup I was not setting up the VM correctly.